cmd/ctr/commands/images/import.go

```diff
@@ -122,6 +122,13 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 				return err
 			}
 		}
+
+		ctx, done, err := client.WithLease(ctx)
+		if err != nil {
+			return err
+		}
+		defer done(ctx)
+
 		imgs, err := client.Import(ctx, r, opts...)
```

Comment:
We seem to already have a lease

https://github.com/containerd/containerd/blob/85309b083714fc8430be7c1a3f8ba0e06f6048a2/import.go#L98

Reply:
The individual calls have it, I am curious if there is something getting garbage collected between import and unpack? Although I would expect if something was getting GC'ed before unpack, it would still end up getting GC'ed after

Reply:
Let me double-check the imported image. If it doesn't have any labels, it could be GC'd before Unpack.